app/newsfeed/server/deinitialize.js

```diff
@@ -0,0 +1,14 @@
+// import { Meteor } from 'meteor/meteor';
+//
+// import { settings } from '../../settings';
+import { deleteRoom } from '../../lib/server/functions';
+import { Rooms } from '../../models';
+
+function deinitializeNewsfeed() {
+	const roomList = Rooms.findByType('n').fetch();
+	roomList.forEach((room) => {
+		deleteRoom(room._id);
```

Comment:
Shouldn't the newsfeed be deactivated (Keeping the rooms, but setting active as false on them) rather than being deinitialised (deleting the rooms)?
In this way, if it is disabled, and then enabled, it retains all of the contents of newsfeeds of users?
@bizzbyster thoughts?

Reply:
Archiving room can be done, but it still retains the data. Isn't disabling newsfeed like not allowing it on the server, so everything should be removed.